src/applications/rated-disabilities/components/NeedHelp.jsx

```diff
@@ -3,12 +3,14 @@ import { CONTACTS } from '@department-of-veterans-affairs/component-library/cont
 
 export default function NeedHelp() {
   return (
-    <va-need-help class="vads-u-margin-y--3">
+    <va-need-help class="vads-u-margin-y--3" uswds="false">
       <div slot="content">
         <p>
-          You can call us at <va-telephone contact={CONTACTS.VA_BENEFITS} />.
+          You can call us at{' '}
+          <va-telephone contact={CONTACTS.VA_BENEFITS} uswds="false" />. We’re
           We’re here Monday through Friday, 8:00 a.m to 9:00 p.m. ET. If you
-          have hearing loss, call <va-telephone contact={CONTACTS['711']} tty />
+          have hearing loss, call{' '}
+          <va-telephone contact={CONTACTS['711']} tty uswds="false" />
```

Comment:
Neither `va-need-help` nor `va-telephone` has a `uswds` property.

Reply:
yes so even though it doesnt have it yet, we still wanted to add it and set it to false since all components are supposed to be updated to V3 by feb 16.